mentoring/programs/mentoring-lead.md

```diff
@@ -0,0 +1,58 @@
+# Internship Mentoring Lead Handbook
+
+## Overview
+
+The Mentoring Lead role is created to help upstream Kubernetes contributors to participates in various mentorship programs like [GSoC](https://github.com/kubernetes/community/blob/master/mentoring/programs/google-summer-of-code.md), [GSoD](https://github.com/kubernetes/community/blob/master/mentoring/programs/google-season-of-docs.md), [LFX Mentorship](https://github.com/kubernetes/community/blob/master/mentoring/programs/lfx-mentorship.md) and many more as an independent organisation or under Cloud Native Computing Foundation(CNCF). 
+
+## Responsibilities
+ The role lead should:
+
+- Keep a calendar of dates for each program (when to submit our project name, when to submit the project scope, when selecting period is, etc) throughout the   year and
+- Coordinate all submissions and participation in each one
```

Comment:
I'd also add a point here about sending out a call for applications and following up with SIGs to ensure that we can get a healthy number of applications.

Reply:
I think it might be helpful to also explicitly call out that they are responsible for following up with mentors to ensure evaluations get submitted on time -- we've had cases where mentors just forgot to do this and we needed to chase them on twitter, etc too 😐